src/ModalContent/ModalContent.stories.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import Modal from '@mui/core/ModalUnstyled';
```

Comment:
```suggestion
import ModalUnstyled from '@mui/core/ModalUnstyled';
```

This might need to be updated